newrelic-integration-e2e/Makefile

```diff
@@ -9,7 +9,6 @@ ROOT_DIR=?
 VERBOSE=?
 LICENSE_KEY=?
 GH_ACTION_DIR=$(CURDIR)/../
```

Comment:
if we run it locally `GH_ACTION_DIR ` can be confusing? By the way if we need files in a lower folder isn't better to place the makefile itself in a lower folder?

Reply:
Moved Makefile to the root and GH_ACTION_DIR is unnecesary